draft-ietf-tls-tls13.md

```diff
@@ -3450,60 +3409,74 @@ the sources from the table above.
   - HkdfLabel.label is "TLS 1.3, " + Label
   - HkdfLabel.hash_value is HashValue.
 
-  1. xSS = HKDF-Extract(0, SS). Note that HKDF-Extract always
-     produces a value the same length as the underlying hash
-     function.
-
-  2. xES = HKDF-Extract(0, ES)
-
-  3. mSS = HKDF-Expand-Label(xSS, "expanded static secret",
-                             handshake_hash, L)
-
-  4. mES = HKDF-Expand-Label(xES, "expanded ephemeral secret",
-                             handshake_hash, L)
-
-  5. master_secret = HKDF-Extract(mSS, mES)
-
-
-  6. traffic_secret_0 = HKDF-Expand-Label(master_secret,
-                                          "traffic secret",
-                                          handshake_hash, L)
-
-  Where handshake_hash includes all messages up through the
-  server CertificateVerify message, but not including any
-  0-RTT handshake messages (the server's Finished is not
-  included because the master_secret is needed to compute
-  the finished key). [[OPEN ISSUE: Should we be including
-  0-RTT handshake messages here and below?.
-  https://github.com/tlswg/tls13-spec/issues/351
-  ]] At this point,
-  SS, ES, xSS, xES, mSS, and mES SHOULD be securely deleted,
-  along with any ephemeral (EC)DH secrets.
-
-  7. resumption_secret = HKDF-Expand-Label(master_secret,
-                                           "resumption master secret"
-                                           handshake_hash, L)
-
-  8. exporter_secret = HKDF-Expand-Label(master_secret,
-                                         "exporter master secret",
-                                         handshake_hash, L)
-
-
-  Where handshake_hash contains the entire handshake up to
-  and including the client's Finished, but not including
-  any 0-RTT handshake messages or post-handshake messages.
-  AT this point master_secret SHOULD be securely deleted.
-~~~
+  Add-Secret(Secret, InputSecret) =
+       HKDF-Extract(HKDF-Expand-Label(Secret, "add secret", "", L),
+                                      InputSecret))
+
+  where L denotes the length of the underlying hash function for
+  HKDF.
+
+  Derive-Secret(Secret, Label, Messages) =
+       HKDF-Extract(HKDF-Expand(Secret, Label, Hash(Messages), L))
```

Comment:
Is HKDF-Extract is defined to take two arguments. Waht is the value of the salting term?